src/main/java/com/car/app/model/dto/CarDto.java

```diff
@@ -12,11 +12,27 @@
 	
 	private Date updatedDate;
 
-	private String brand;
-	
     private Date registration;
 	
-	private String country;
+	private String CountryName;
```

Comment:
Why did you change the attribute names ? Changing this breaks the API definition, meaning that a client of your API would need to change its implementation.